letsencrypt/README.rst

```diff
@@ -13,7 +13,7 @@ Installation
 ============
 
 After installation, this module generates a private key for your account at
-letsencrypt.org automatically in ``$data_dir/letsencrypt/account.key``. If you
+letsencrypt.org automatically in ``$data_dir/letsencrypt/account_key``. If you
```

Comment:
Well, I had changed the code to save it as `account_key` because extensions do not make sense in the *nix world from what I remember. If the user wants to figure out what the `account_key` is all about they can `file account_key`. It's sort of redundant information I understand.